contracts/Stoppable.sol

```diff
@@ -0,0 +1,38 @@
+pragma solidity 0.5.10;
+
+import './Owned.sol';
+
+contract Stoppable is Owned {
+    
+    bool isRunning;
```

Comment:
Are you inviting the developer of the child contract to change the value at will?

Reply:
Indeed, I prefer this design pattern than a real kill switch

Reply:
What I mean, is which way do you want the developer of the child contract to use:

```solidity
isRunning = false;
```
or

```solidity
pauseContract();
```
One emits an event and controls for ownership, the other doesn't.

Reply:
I meant `bool private isRunning;`